cypress/e2e/smoke/Wallets/derivxResetpassword.cy.js

```diff
@@ -0,0 +1,155 @@
+import '@testing-library/cypress/add-commands'
+function clickAddDerivxButton() {
+  cy.get('.wallets-available-dxtrade__icon')
+    .parent('.wallets-trading-account-card')
+    .click()
+}
+
+function verifyDerivxCreation(accountType) {
+  let expectedText
+  if (accountType === 'Real') {
+    expectedText = 'Create a Deriv X password'
+    cy.get('div').contains(expectedText).should('be.visible')
```

Comment:
Why the need to use declare a variable for this when it's not being used elsewhere? Why not just:
```suggestion
    cy.get('div').contains('Create a Deriv X password').should('be.visible')
```